tools/pipeline-runner/cloudbuild.yaml

```diff
@@ -32,35 +36,41 @@ steps:
       - './tools/pipeline-runner/Dockerfile'
       - '-t'
       - 'gcr.io/$PROJECT_ID/devrel-pipeline:latest'
+      - '--cache-from'
+      - 'gcr.io/$PROJECT_ID/devrel-pipeline:latest'
       - './tools/pipeline-runner'
   - name: 'gcr.io/$PROJECT_ID/devrel-pipeline:latest'
-    id: 'Load Secrets'
-    entrypoint: "sh"
+    id: 'Cleanup Org'
+    entrypoint: "bash"
     args:
       - "-c"
       - |-
-        echo "export APIGEE_PASS=\"$(gcloud secrets versions access latest \
-          --secret=devrel_apigee_pass --format="get(payload.data)" | \
-          base64 -d)\"" >> env.txt
-
-        echo "export APIGEE_USER=\"$(gcloud secrets versions access latest \
-          --secret=devrel_apigee_user --format="get(payload.data)" | \
-          base64 -d)\"" >> env.txt
-
-        echo "export GITHUB_TOKEN=\"$(gcloud secrets versions access latest \
-          --secret=devrel_github_token --format="get(payload.data)" | \
-          base64 -d )\"" >> env.txt
+        if [ "$_CLEAN_ORG" = "true" ]; then
+          APIGEE_TOKEN=$(gcloud auth print-access-token)
+          PATH=$$PATH:"$$PWD/tools/apigee-sackmesser/bin"
+          sackmesser clean all --googleapi -t "$$APIGEE_TOKEN" -o "$_APIGEE_X_ORG" --quiet
+          sackmesser clean all --apigeeapi -u "$$APIGEE_USER" -p "$$APIGEE_PASS" -o "$_APIGEE_ORG" --quiet
+        fi
+    env:
+      - 'APIGEE_ORG=$_APIGEE_ORG'
+      - 'APIGEE_X_ORG=$_APIGEE_X_ORG'
+    secretEnv:
+      - 'APIGEE_USER'
+      - 'APIGEE_PASS'
   - name: 'gcr.io/$PROJECT_ID/devrel-pipeline:latest'
     id: 'Run Pipeline'
     entrypoint: "bash"
     args:
       - "-c"
       - |-
-        max_duration=7080 # cloud build timeout minus 2m
-        source env.txt && \
+        max_duration=6960 # cloud build timeout minus 4m
         if [ -n "$_PR_NUMBER" ]; then
+          FULL_PIPELINE_REQUIRED=$(curl "https://api.github.com/repos/$_REPO_GH_ISSUE/issues/$_PR_NUMBER" | jq '.body | contains("[X] PR requires full pipeline run")' )
```

Comment:
There's a possibility that the dependency on the exact string match might be a bit brittle, but I see no quick/easy drop-in alternative. I acknlowedge the updated PR template and the advisory within.

We can see how it goes. Impact of condition not matching isn't problematic anyways, so this is fine.